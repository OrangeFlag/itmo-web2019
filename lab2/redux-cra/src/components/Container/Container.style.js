import styled from "styled-components";

const Container = styled("div")`
  display: flex;
  flex-flow: row nowrap;
  align-items: center;

  width: 76%;

  background-color: #0138a5;
  background-color: ___CSS_1___;

  color: #ffffff;
  color: ___CSS_2___;
  
  padding: 12px 10%;
  padding-top: ___CSS_3___;
  padding-bottom: ___CSS_4___;
  
  box-shadow: 0px 2px 2px 0px rgba(0, 0, 0, 0.30);
  margin: 7px;


  * {
    margin-right: 16px;
  }
`;

export default Container