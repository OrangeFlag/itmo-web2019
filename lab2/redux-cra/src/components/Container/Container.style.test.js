import React from 'react';
import {shallow} from 'enzyme';
import Container from './Container.style';

describe('Container tests', () => {
    it('should render correctly +', () => {
        const container = shallow(<Container>+</Container>);

        expect(container).toMatchSnapshot();
    });
});