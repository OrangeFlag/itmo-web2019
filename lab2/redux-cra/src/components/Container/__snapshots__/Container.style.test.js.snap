// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container tests should render correctly + 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  display: flex;
  flex-flow: row wrap;
  align-items: center;

  width: 76%;

  background-color: #0138a5;
  background-color: ___CSS_1___;

  color: #ffffff;
  color: ___CSS_2___;
  
  padding: 12px 10%;
  padding-top: ___CSS_3___;
  padding-bottom: ___CSS_4___;
  
  box-shadow: 0px 2px 2px 0px rgba(0, 0, 0, 0.30);
  margin: 7px;


  * {
    margin-right: 16px;
  }
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "usesTheme": false,
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  +
</StyledComponent>
`;
